Backend/AlgorithmsManager.py

```diff
@@ -306,14 +306,14 @@ def add_trends_snapshot_to_database(self, snap):
     def get_trends_snapshot_from_database(self, trends, fromdate, todate=datetime.now(), woeid=1):
         querytemplate = "SELECT * FROM trends_snapshot " \
                         "WHERE created_date >= '{}' AND created_date < '{}' " \
-                        "AND trend_content = '{}'" \   
-                        "{}" \
-                        "ORDER BY id ASC;"
```

Comment:
Why is this deleting the order by clause? I'm pretty sure the order by clause matters